### ISSUE TITLE
Error Messages while running 'vnx' command for first time

### ISSUE THREAD

**Issue by sunnyrarora** [emc-openstack/storops#17]
Hi Team,
I am facing an weird  issue that whenever i run the command 'vnx' for first time OR getting any information by it for example "vnx.model"  throws some error messages on console and Next time by running same command  gives me proper output till the session is running. 
am i missing anything ? Is there any other way to pull basic details like serial no. , model and name of VNX ?
```
[root@Test ~]# python2.7
Python 2.7.10 (default, May 27 2016, 05:55:08)
[GCC 4.4.7 20120313 (Red Hat 4.4.7-17)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from storops import VNXSystem
>>> import logging
>>> logging.basicConfig(level=logging.WARNING)
>>> vnx = VNXSystem('10.xx.xx.xx', 'test_user', 'password')
>>> vnx
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/storops/lib/common.py", line 50, in __repr__
    return self.json(indent=4)
  File "/usr/local/lib/python2.7/site-packages/storops/lib/common.py", line 47, in json
    return json.dumps(self.get_dict_repr(), indent=indent, sort_keys=True)
  File "/usr/local/lib/python2.7/site-packages/storops/lib/common.py", line 42, in get_dict_repr
    props = self._get_properties(dec)
  File "/usr/local/lib/python2.7/site-packages/storops/lib/resource.py", line 155, in _get_properties
    value = getattr(self, name)
  File "/usr/local/lib/python2.7/site-packages/storops/lib/resource.py", line 184, in __getattr__
    ret = self._get_property_from_raw(item)
  File "/usr/local/lib/python2.7/site-packages/storops/vnx/resource/__init__.py", line 70, in _get_property_from_raw
    value = super(VNXCliResource, self)._get_property_from_raw(item)
  File "/usr/local/lib/python2.7/site-packages/storops/lib/resource.py", line 197, in _get_property_from_raw
    self.update()
  File "/usr/local/lib/python2.7/site-packages/storops/vnx/resource/system.py", line 144, in update
    self.update_nodes_ip()
  File "/usr/local/lib/python2.7/site-packages/storops/vnx/resource/system.py", line 139, in update_nodes_ip
    self._cli.set_ip(self.spa_ip, self.spb_ip, self._get_cs_ip())
  File "/usr/local/lib/python2.7/site-packages/storops/vnx/resource/system.py", line 183, in _get_cs_ip
    return VNXDomainNodeList.get_cs_ip(self.serial, self._cli)
  File "/usr/local/lib/python2.7/site-packages/storops/vnx/resource/vnx_domain.py", line 83, in get_cs_ip
    node = dnl.get_node(serial)
  File "/usr/local/lib/python2.7/site-packages/storops/vnx/resource/vnx_domain.py", line 76, in get_node
    'domain node "{}" not found'.format(node_id))
storops.exception.VNXObjectNotFound: domain node "APM00XXXXXXXX" not found
```

Next time running the same command gives proper output.

```
>>> vnx
{
    "VNXSystem": {
        "agent_rev": "7.33.9 (1.5)",
        "existed": true,
        "hash": 8733XXXXX,
        "model": "VNX5600",
        "model_type": "Rackmount",
        "name": "X",
        "revision": "05.33.009.5",
        "serial": "APM00XXXXXXXX"
    }
}

```

Thanks
Regards

Sunny

**Comment by jealous** [comment]
Hi Sunny,

The root cause is probably that VNX is not added to any VNX domain.  I have modified the code to swallow this exception and log an info instead.  It will affect the retrieve of control station IP if domain is not available.  Because we don't have any other ways to retrieve the CS IP from naviseccli.  If you are using a block only system, feel free to ignore this error.

The fix will be included in 0.2.16.

**Comment by jealous** [comment]
I am closing this issue because the fix has been merged.  Please feel free to re-open it if the issue occurs again.
